oci/spec_opts_test.go

```diff
@@ -0,0 +1,84 @@
+package oci
+
+import (
+	"context"
+	"testing"
+
+	"github.com/containerd/containerd/containers"
+	"github.com/containerd/containerd/namespaces"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func TestSpecWithEnvAppend(t *testing.T) {
```

Comment:
nit: If this is testing `WithEnv`, it should be `TestWithEnv`. I think all the branches can be tested in one test case by using different env variables. `Spec` is already implied by the package name.